src/components/_organisms/Table/Table.tsx

```diff
@@ -1,11 +1,14 @@
 import { Table } from 'antd'
 import cn from 'classnames'
-import { columns } from 'src/constants'
+import { Columns } from 'src/constants'
 
 import { TableType } from './Table.d'
 
-export const List = ({ className, data }: TableType): JSX.Element => (
-  <div className={cn('user-list', className)}>
-    <Table dataSource={data} size="middle" columns={columns} bordered />
-  </div>
-)
+export const List = ({ className, data }: TableType) => {
+  const columns = Columns()
```

Comment:
it is not reusable if you put Columns here
Table is just component and you need to reuse it in different places with different columns